src/driver/mod.rs

```diff
@@ -312,14 +312,6 @@ impl Default for Driver {
     }
 }
 
-impl Drop for Driver {
-    /// Leaves the current connected voice channel, if connected to one, and
-    /// forgets all configurations relevant to this Handler.
-    fn drop(&mut self) {
-        drop(self.sender.send(CoreMessage::Poison));
-    }
```

Comment:
I have a gut feeling this might still be necessary due to `Interconnect` effectively creating a reference loop on `Sender<CoreMessage>`. Can you verify that all threads and tasks actually die?

Reply:
Ah, this might be why my thread count looks a little higher than normal, reverting that